Appearances/Base.lua

```diff
@@ -96,11 +110,16 @@ function SetCollector:IncludeSet(collection, uid, setID, armorType, class, facti
             Location = description,
             Variants = { SetCollector:IncludeVariant(setID, setInfo) }
         }
+
+        SetCollector.db.global.setMap["SET " .. setID] = {setID = setID, collection = collection.Description, baseSetId = setID}
```

Comment:
I like the concept. I have been wanting to explore a way to provide additional search/filtering options around the existing Blizzard defined sets while allowing for Set Collector and user-defined sets.

Reply:
We'll want to keep an eye on memory usage. I have the expansions split out allowing for users to exclude expansions they don't care about to keep memory usage down as the game introduces more expansions and sets

Reply:
> We'll want to keep an eye on memory usage. I have the expansions split out allowing for users to exclude expansions they don't care about to keep memory usage down as the game introduces more expansions and sets

Yeah, the memory usage has been on the backburner, my code is horrible about it right now as I try to figure out what info is important.  Also, if I knew what I actually wanted to do, that would help me figure out what info is important.  I think I'm on "Plan F" now, after a couple false starts and realizing plans D and E didn't go far enough.

Reply:
> I like the concept. I have been wanting to explore a way to provide additional search/filtering options around the existing Blizzard defined sets while allowing for Set Collector and user-defined sets.

Yeah.  The Blizzard UI is nice and helpful, but not (afaik) expandable.  Ultimately, I'd like to add a search/filter bar like the Blizzard UI has, and then add all the Ensembles: X in the game that aren't (for some crazy reason) flagged as a cosmetic set.  We're going to need more categories!